google/cloud/datastore/batch.py

```diff
@@ -290,7 +290,7 @@ def _assign_entity_to_pb(entity_pb, entity):
 
     Helper method for ``Batch.put``.
 
-    :type entity_pb: :class:`google.cloud.datastore._generated.entity_pb2.Entity`
+    :type entity_pb: :class:`.datastore._generated.entity_pb2.Entity`
```

Comment:
This doesn't look right:  it should be fully-qualified, or else `._generated....`.